Sources/Swift/Protocol/Codable/SentryBreadcrumbCodable.swift

```diff
@@ -1,7 +1,16 @@
 @_implementationOnly import _SentryPrivate
 import Foundation
 
-extension Breadcrumb: Decodable {
+#if SENTRY_SWIFT_PACKAGE
```

Comment:
`m`: We should document somewhere in this repository why this approach with `#if-else`  is used, so we can easily understand it again when revisiting it in a couple of years.

Reply:
Maybe a page in the `develop-docs` directory would be the best place to fully explain it, but then we add a comment to each of the decodable helper classes with a reference.